generators/server/templates/src/main/resources/config/application-dev.yml.ejs

```diff
@@ -203,6 +203,8 @@ spring:
 
 server:
   port: <%= serverPort %>
+  # make sure requests the proxy uri instead of the server one
```

Comment:
```suggestion
  # Use the proxy URI from webpack instead of the default from Spring Security
```

Reply:
It's broader than spring security, more spring-web. But it will be used by Spring security in particular